identity-service/src/routes/push_notifications.js

```diff
@@ -0,0 +1,101 @@
+const {
+  handleResponse,
+  successResponse,
+  errorResponseBadRequest,
+  errorResponseServerError
+} = require('../apiHelpers')
+const models = require('../models')
+const config = require('../config')
+const { createPlatformEndpoint } = require('../awsSNS')
+
+const iOSSNSParams = {
+  PlatformApplicationArn: config.get('awsSNSiOSARN')
+}
+// const androidSNSParams = {
+//   PlatformApplicationArn: config.get('awsSNSAndroidARN')
+// }
+
+const DEVICE_TYPES = new Set(['ios', 'android'])
+
+module.exports = function (app) {
+  /**
+   * Create or update mobile push notification settings
+   * POST body contains {userId, settings: {favorites, milestonesAndAchievements, reposts, announcements, followers}}
+   */
+  app.post('/push_notifications/settings', handleResponse(async (req, res, next) => {
+    const { userId, settings } = req.body
```

Comment:
Shouldn't we do the same thing here that we do for /notifications/settings where we use a signed piece of info (with auth) to extract user id rather than having it passed in?

https://github.com/AudiusProject/audius-protocol/blob/master/identity-service/src/routes/notifications.js#L446

Reply:
@raymondjacobson i have this item on the to do checklist in the card on trello. good catch!